### PR TITLE
Origin/content/docupdates feb21

### PR DESCRIPTION
**Pull request by RadhaV2020**

### CODE REVIEW

File: source/requirements/requirements.rst [hunk 1]
```diff
@@ -23,6 +23,10 @@ All IBM z/OS collections require these versions of software:
 * `Ansible version`_: 2.9 or later
 * `Python`_: 2.7 or later
 * `OpenSSH`_
+   * Modules which can have Ansible tasks delegated to a localhost need not
+     have OpenSSH enabled. Such modules often rely on a REST API for automation.
+     Refer to the collections under `managed node documentation`_
```
Comment (ddimatos) [inline source/requirements/requirements.rst:28]: I don't see a URL for this link
image

File: source/installation/installation.rst [hunk 2]
```diff
@@ -43,67 +43,75 @@ Ansible Galaxy
...
    $ ansible-galaxy collection install <namespace>.<collection name>
 
-Here is an example of installing a collection:
+You can install an Ansible collection in one of these ways:
+
+Fresh installation
+------------------
+By default, the `ansible-galaxy`_ command installs the latest available
+collection. If you would like to install a previous version of a collection or if
+the installation is unsuccessful, use the ``--force option to override an existing
```
Comment (ddimatos) [inline source/installation/installation.rst:65]: Missing closure after --force , ``--force``

**Review by ddimatos — CHANGES_REQUESTED** [review]
Thanks, changes look good, I see two errors needing correction, I made comments.

File: source/requirements/requirements.rst (hunk 1 above)
Comment (RadhaV2020) [reply source/requirements/requirements.rst]: This was intended to be just a highlight. Fixed it

**Review by ddimatos — APPROVED** [review]

**Pull request merged**